src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -33,7 +33,7 @@ export const useInfiniteTransactionList = ({
   const { currentAddress: address } = useCurrentAddressStore();
   const { currentCurrency: currency } = useCurrentCurrencyStore();
   const pendingTransactions = usePendingTransactionsStore(
-    (s) => s.pendingTransactions[address],
+    (s) => s.pendingTransactions[address] ?? [],
```

Comment:
I thought about this when doing this change, thought it was fine to leave as undefined

since it's not, and I'm not sure if I think it's a beautiful solution yet, instead of recreating the array on every selector run, we could return a stable empty array 

```ts
const stableEmptyPendingTransactionsArray = []
export const useInfiniteTransactionList = ({ ... }) =>
  ...
  const pendingTransactions = usePendingTransactionsStore(
      (s) => s.pendingTransactions[address] || stableEmptyPendingTransactionsArray,
   );
```
tbh I dont think I like how this reads, the way I'd have to jump array to look what this `stableEmptyPendingTransactionsArray` means/is, but as of rn is the best way I think to not need unnecessary rerenders

this way we can use `pendingTransactions` in the dependency array at `const formattedTransactions = useMemo(` without rerunning this memo every render